data/default/ACCESSCONTROL-ROLEACTIONS/roleactions.json

```diff
@@ -14095,6 +14095,24 @@
       "actionid": 1980,
       "actioncode": "",
       "tenantId": "default"
+    },
+    {
```

Comment:
Please provide access to beneficiary endpoints to registrars and distributors as well